mypy.ini

```diff
@@ -62,6 +62,9 @@ ignore_errors = False
 [mypy-torch.distributed.distributed_c10d.*]
 ignore_errors = False
 
+[mypy-torch.distributed.nn.*]
+ignore_errors = False
```

Comment:
I am not 100% sure how this usually work on ghstack - do you have to wait for all 4 PRs to be accepted and commit in order? (seems like if all PR merged together you can just remove this?)

Reply:
Yes. After all 4 PRs merged together, this line (and all other torch.distributed.* related lines) will be removed.